process faults {7: ({6, 5}, 3043267948)}
network faults {4: [{1, 0, 6, 3}, {2}, {4}, {5}], 5: [{2, 1, 0, 3}, {4}, {6, 5}]}
{6: ("C031963E58A3BD6FF77FA3C21A5113534071434085ADBB7185A21574B1F8A9B3", {1, 4, 3, 0, 5, 2, 6}), 9: ("4DF6BDF3F36FADF62D43EB851DE2310386E60D17A41007D51CF5CD08378D58E3", {1, 0, 4, 6, 3, 5, 2}), 10: ("8D661C0B195AC730C37C7DD1DED6CEED7B344964390813A7BC95AFD52EBB2284", {0, 2, 4, 3, 5, 6, 1}), 5: ("A30388384186FB0A7E86E1D9D2377999D7A3F63432873D5E8CA75DD1078B9DC7", {0, 5, 6, 1, 4, 3, 2}), 7: ("9FC2F2E9E93734947537409DB0716E61061B964624F68A53AD46102B96F31564", {2, 1, 0, 3, 6, 5, 4}), 8: ("2E083FEE2BFF4CEDD58A054AB38F94F093DAA3ECA8A6DDA77B0F2C11BBAE9440", {1, 3, 4, 0, 2, 5, 6})}
7
reason: all committed
done!
